core_screenshot_config.yaml

```diff
@@ -110,6 +110,18 @@ secondary:
       page.done();
     message: waiting 30 sec to load AR secondary
 
+  DE:
+    renderSettings:
+      maxWait: 60000
+    overseerScript: >
+      page.manualWait(); 
+      await page.waitForSelector("#tab_1[href='#test_tabs_1_content']");
+      await page.waitForDelay(5000); 
```

Comment:
I don't think the wait before the click is needed. @muamichali would you be ok with removing it? Less waiting will reduce the $ cost of each screenshot and reduce the chances of http 413 errors.